### PR TITLE
Update readme to fix or remove invalid links

### PR DESCRIPTION
**Pull request by pozsgaic**
Signed-off-by: pozsgaic <chris.pozsgai@progress.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
-<img src="https://github.com/habitat-sh/habitat/blob/master/www/source/images/habitat-logo.png" width="200">
+<img src="images/chef-logo.svg">
```
Comment (mwrock) [inline README.md:1]: I would remove this entirely. I renders really large but none of our other product repos have a logo.

File: README.md [hunk 2]
```diff
@@ -45,30 +44,25 @@ Graphics that will help you and your team better understand the concepts and how
...
-* [Architecture Overview](https://github.com/habitat-sh/habitat/raw/master/www/source/images/infographics/habitat-architecture-overview.png)
-* [Initial Package Build Flow](https://github.com/habitat-sh/habitat/raw/master/www/source/images/infographics/habitat-initial-package-build-flow.png)
-* [Application Rebuild Flow](https://github.com/habitat-sh/habitat/raw/master/www/source/images/infographics/habitat-application-rebuild-flow.png)
-* [Dependency Update Flow](https://github.com/habitat-sh/habitat/raw/master/www/source/images/infographics/habitat-dependency-update-flow.png)
-* [Promote Packages Through Channels](https://github.com/habitat-sh/habitat/raw/master/www/source/images/infographics/habitat-promote-packages-through-channels.png)
+* [Architecture Overview](images/habitat-architecture-overview.png)
+* [Initial Package Build Flow](images/habitat-initial-package-build-flow.png)
+* [Application Rebuild Flow](images/habitat-application-rebuild-flow.png)
+* [Dependency Update Flow](images/habitat-dependency-update-flow.png)
+* [Promote Packages Through Channels](images/habitat-promote-packages-through-channels.png)
```
Comment (mwrock) [inline README.md:51]: All of the above links should be "raw" links otherwise the image will appear in the github view and be cut off. This probably means you need to commit the images before this PR is merged so you have access to those raw links. Go ahead and create a separate pr just for these images and we can get that merged asap.

File: README.md [hunk 3]
```diff
@@ -45,30 +44,25 @@ Graphics that will help you and your team better understand the concepts and how
...
 
 ### Habitat and **Docker**
-* [Initial Docker Container Publishing Flow](https://github.com/habitat-sh/habitat/raw/master/www/source/images/infographics/habitat-initial-docker-container-publishing-flow.png)
-* [Automated Docker Container Publishing Flow](https://github.com/habitat-sh/habitat/raw/master/www/source/images/infographics/habitat-automated-docker-container-publishing-flow.png)
+* [Initial Docker Container Publishing Flow](images/habitat-initial-docker-container-publishing-flow.png)
+* [Automated Docker Container Publishing Flow](images/habitat-automated-docker-container-publishing-flow.png)
```
Comment (mwrock) [inline README.md:55]: See above comment for these images as well.

File: README.md [hunk 4]
```diff
@@ -126,9 +120,9 @@ The Habitat core team's project tracker is also public and on [Github.](https://
 
 ## Community and support
 
-* [Habitat Slack](http://slack.habitat.sh)
+* [Chef Community Slack](https://community-slack.chef.io/)
 * [Forums](https://forums.habitat.sh)
-* Community triage is every Tuesday at 10am Pacific. The link to participate is shared in the [Habitat Slack channel](http://slack.habitat.sh), and videos are posted on the [Habitat YouTube channel](https://youtube.com/channel/UC0wJZeP2dfPZaDUPgvpVpSg).
+* Community triage is every Thursday at 9am Pacific.  More information can be found in the Connect section of [Chef Community](https://community.chef.io/)
```
Comment (mwrock) [inline README.md:125]: `Community triage` should be `The Chef Community meeting`

**Review by mwrock — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (pozsgaic) [reply README.md:51]: Will do.

**Review by mwrock — APPROVED** [review]
